src/pages/gen2/start/quickstart/nextjs-app-router-server-components/index.mdx

```diff
@@ -0,0 +1,351 @@
+export const meta = {
+  title: 'Next.js App Router (Server Components)',
+  description: 'Get started with AWS Amplify (Gen 2) using the Next.js App Router using Server Components.'
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+This Quickstart guide will walk you through how to build a task list application with TypeScript, Next.js **App Router with Server Components**, and React. If you are new to these technologies, we recommend you go through the official [React](https://react.dev/learn/tutorial-tic-tac-toe), [Next.js](https://nextjs.org/docs/getting-started/installation), and [TypeScript](https://www.typescriptlang.org/docs/handbook/typescript-from-scratch.html) tutorials first.
+
+import prerequisites from 'src/fragments/gen2/quickstart/prerequisites.mdx';
```

Comment:
Non blocking future thought for fragment usage, maybe we should just export a component that contains the import and Fragment itself, so we could import these with easier to understand component names.